cmd/crane/cmd/mutate.go

```diff
@@ -119,20 +134,28 @@ func NewCmdMutate(options *[]crane.Option) *cobra.Command {
 	}
 	mutateCmd.Flags().StringSliceVarP(&anntns, "annotation", "a", nil, "New annotations to add")
 	mutateCmd.Flags().StringSliceVarP(&lbls, "label", "l", nil, "New labels to add")
-	mutateCmd.Flags().StringVar(&entrypoint, "entrypoint", "", "New entrypoing to set")
+	mutateCmd.Flags().StringVar(&entrypoint, "entrypoint", "", "New entrypoint to set")
+	mutateCmd.Flags().BoolVarP(&removeMode, "remove", "r", false, "Enable remove mode")
 	mutateCmd.Flags().StringVarP(&newRef, "tag", "t", "", "New tag to apply to mutated image. If not provided, push by digest to the original image repository.")
 	return mutateCmd
 }
 
 // splitKeyVals splits key value pairs which is in form hello=world
-func splitKeyVals(kvPairs []string) (map[string]string, error) {
+// if requireSplit set true, only gathers keys otherwise.
+func splitKeyVals(kvPairs []string, enforceSplit bool) (map[string]string, error) {
```

Comment:
Instead of repurposing this method, and making it do two things depending on `enforceSplit`, maybe let's just have a separate method that translates a `[]string` -> `map[string]string` (or `map[string]struct{}`, even).